runners/flink/flink_runner.gradle

```diff
@@ -151,6 +151,7 @@ dependencies {
   implementation "org.apache.flink:flink-streaming-java_2.11:$flink_version"
   // RocksDB state backend (included in the Flink distribution)
   provided "org.apache.flink:flink-statebackend-rocksdb_2.11:$flink_version"
+  testImplementation "org.apache.flink:flink-statebackend-rocksdb_2.11:$flink_version"
```

Comment:
Does the test code reference classes from this dep (ditto for other deps added this way)?

During test execution we should get `testRuntimeClasspath -> testRuntimeOnly -> runtimeOnly -> provided`.

You can quickly check to see if compilation is satisfied with `./gradlew testCompileJava`

Reply:
Mostly going off where they were added in https://github.com/apache/beam/commit/2e72638c6b08631bbb9a29e6c24f97043ffd0a31 where @dpcollins-google didn't add them everywhere - I'll go ahead and confirm, and re-request review when I've confirmed

Reply:
Edited last commit